### PR TITLE
Replacing sync steps

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2372

@dariomws wanna take a look?

### CODE REVIEW

File: Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-1-10-in-exchange-online.md [hunk 1]
```diff
@@ -229,20 +229,12 @@ If you have a hybrid configuration with an Microsoft 365 or Office 365 connector
 
 If you have a hybrid configuration and the recipient is located in the on-premises Exchange organization, it's possible that the recipient's email address isn't properly synchronized with Microsoft 365 or Office 365. Follow these steps to synchronize directories manually:
 
-1. Log into the on-premises server that's running the Directory Synchronization Tools.
+1. Log into the on-premises server that's running Azure AD Connect sync.
```
Comment (dariomws) [inline Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-1-10-in-exchange-online.md:232]:
```suggestion
1. Log into the on-premises server that's running Azure AD Connect.
```

I think it's ok. Maybe it looks better without the word sync but just a small suggestion.

**Review by dariomws — APPROVED** [review]

File: Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-1-10-in-exchange-online.md (hunk 1 above)
Comment (get-itips) [reply Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-1-10-in-exchange-online.md:232]: To my surprise, seems to be the full name of it, look:
https://docs.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-sync-whatis

Comment (dariomws) [reply Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-1-10-in-exchange-online.md:232]: Azure AD Connect is the formal and full name of the tool I think:
https://www.microsoft.com/download/details.aspx?id=47594
You can also see it when you install it, and this is why I think the first letter of the word "sync" is lower case:
image

Anyway seeing the article you sent and similar ones I think you can use it. You can add the word sync, synchronization services, sync service, sync tool, sync engine, etc.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**